devtools/etdump/utils.h

```diff
@@ -0,0 +1,34 @@
+// (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
+
+#include <cstddef>
+#include <cstdint>
+
+#pragma once
+
+namespace executorch {
+namespace etdump {
+namespace internal {
+
+/**
+ * Aligns a pointer to the next multiple of `alignment`. `alignment` must be a
+ * power of 2.
+ *
+ * @param[in] ptr Pointer to align.
+ * @param[in] alignment Alignment to align to. Must be a power of 2.
+ *
+ * @returns A pointer aligned to `alignment`.
+ */
+
+inline uint8_t* align_pointer(void* ptr, size_t alignment) {
```

Comment:
- No need to mention the power of 2 detail in the top message since it's on the param line
- Doxygen comments should not have blank lines after them

```suggestion
/**
 * Aligns a pointer to the next multiple of `alignment`.
 *
 * @param[in] ptr Pointer to align.
 * @param[in] alignment Alignment to align to. Must be a power of 2.
 *
 * @returns A pointer aligned to `alignment`.
 */
inline uint8_t* align_pointer(void* ptr, size_t alignment) {
```